app/src/main/java/com/example/democicdgithub/MainActivity.kt

```diff
@@ -4,8 +4,11 @@ import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 
 class MainActivity : AppCompatActivity() {
+    companion object {
+        const val abc= 123
+    }
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
     }
-}
+}
```

Comment:
<table data-meta="generated_by_check_style">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/toandn0696/DemoGithubAction/blob/adc77ffd7e0b8b58eda8fc9dfab0067aa9ea6158/app/src/main/java/com/example/democicdgithub/MainActivity.kt#L14"></span>The file /home/runner/work/DemoGithubAction/DemoGithubAction/app/src/main/java/com/example/democicdgithub/MainActivity.kt is not ending with a new line.</td>
    </tr>
  </tbody>
</table>